source/guides/recipes/post-build-hook.md

```diff
@@ -28,18 +27,20 @@ $ cat /etc/nix/key.public
 example-nix-cache-1:1/cKDz3QCCOmwcztD2eV6Coggp6rqc9DGjWv7C0G+rM=
 ```
 
-Then update [`nix.conf`](https://nixos.org/manual/nix/stable/command-ref/conf-file.html) on any machine that will access the cache.
-Add the cache URL to [`substituters`](https://nixos.org/manual/nix/stable/command-ref/conf-file.html#conf-substituters) and the public key to [`trusted-public-keys`](https://nixos.org/manual/nix/stable/command-ref/conf-file.html#conf-trusted-public-keys):
+[](custom-binary-cache) on any machine that will access the bucket.
```

Comment:
I think some context was accidentally left out here. `[](custom-binary-cache)` doesn't show up because `[]` is empty.

Reply:
Same as my other comment.
I have not setup a binary cache before so I am having trouble understanding what commands are run on which machine. Is this correct?

```suggestion
Any machine that will access the bucket, must have knowledge of the cache URL and public key of the binary cache machine.
```

A little redundant but more explicit. I'm not emotionally tied to my suggestion so feel free to modify.

Reply:
The code is being a bit clever here, MyST will replace `[](reference)` with the heading at the reference target (added in  `add-binary-cache.md`). You have to build the site to see the result.